message.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime, timedelta
+import pytz, uuid
```

Comment:
One module per import.

```
import pytz
import uuid
```

The datetime line is okay, though, because both imports are from the same module.